resources/istio/templates/istio-proxy-reset.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-proxy-reset
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istio-proxy-reset
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["replicasets", "deployments", "daemonsets", "statefulsets"]
+    verbs: ["get", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-proxy-reset
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio-proxy-reset
+subjects:
+  - kind: ServiceAccount
+    name: istio-proxy-reset
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: istio-proxy-reset
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook-delete-policy: before-hook-creation
+    helm.sh/hook: post-upgrade
+    helm.sh/hook-weight: "20"
+spec:
+  backoffLimit: 1
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: istio-proxy-reset
+      restartPolicy: Never
+      containers:
+        - name: proxy-reset
+          image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20200617-32c1f3ff
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "TEMPORARY - NEEDS TO BE REMOVED BEFORE MERGE"
+              
+              kubectl delete po --all -n applicationbrokerupgradetest
```

Comment:
note to self: this needs to be done only on our ci, customer will NOT have these namespaces